setup.py

```diff
@@ -26,10 +26,12 @@
     name='simanneal',
     version=version,
     description='Simulated Annealing in Python',
-    license='BSD',
+    license='ISC',
     author='Matthew Perry',
     author_email='perrygeo@gmail.com',
     url='https://github.com/perrygeo/simanneal',
     long_description=LONG_DESCRIPTION,
-    packages=['simanneal'],
-    install_requires=[])
+    packages=['simanneal', 'tests'],
```

Comment:
This will "pollute" the global namespace with a package named `tests`. If we're going to include tests, we need to put tests as a submodule of the simanneal package.